### DISCUSSION TITLE
Improve HA update end user experience - "supervisor needs to be updated first" error

### DISCUSSION THREAD

**Discussion by PavelKuzub** [home-assistant/frontend#18117] [Configuration]
Hello, end-user experience of the update process may be improved if updates that depend on updating other components first are not offered or greyed out with description on what has to be updated before first.

For instance, Home Assistant Supervisor has to be updated before Home Assistant Core. 
I learned it because that's what the error message said after I attempted to install HA Core update. 
UI should not have offered me to update HA Core when HA Supervisor update is available and it is pre-requisite.
Especially when it is displayed as the first item in the update list.

![image](https://github.com/home-assistant/frontend/assets/186876/30527ee6-96f7-400c-af4c-50483b7a06b1)

**Comment by frenck** [comment]
This is not a frontend suggestion or issue. Please use our community forums to raise this feature request.

[Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

Thanks! 👍

../Frenck

**Reply by PavelKuzub** [reply]
Thanks Frenck, will raise there
